CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 22.02+ (???)
 ------------------------------------------------------------------------
 - Feature: [#1327] Readd the game intro (use commandline switch --intro to enable)
+- Feature: [#1345] Show vehicle obsolete date in build vehicle window.
```

Comment:
What does 1345 have to do with this? Think you meant 1350

Reply:
maybe just a fat finger typo :|